src/sdk/pynni/nni/compression/torch/pruning/amc/channel_pruning_env.py

```diff
@@ -200,7 +200,9 @@ def step(self, action):
                 self.best_reward = reward
                 self.best_strategy = self.strategy.copy()
                 self.best_d_prime_list = self.d_prime_list.copy()
-                torch.save(self.model.state_dict(), os.path.join(self.args.output, 'best_wrapped_model.pth'))
+                best_model = os.path.join(self.args.output, 'best_model.pth')
+                best_mask = os.path.join(self.args.output, 'best_mask.pth')
+                self.pruner.export_model(model_path=best_model, mask_path=best_mask)
```

Comment:
do we have ut or it to test `export_model`?

Reply:
Thanks, ut added.